spec/factories/clients.rb

```diff
@@ -34,7 +34,16 @@
 FactoryBot.define do
   factory :client do
     trait :with_return do
-      tax_returns { FactoryBot.create_list(:tax_return, 1, status: 101) }
+      transient do
+        status { "intake_in_progress" }
+      end
+      after(:create) do |client, evaluator|
+        create :tax_return, client: client, status: evaluator.status
+      end
+    end
+
+    factory :client_with_status do
```

Comment:
I also see that potentially this factory is so named because it indicates that you're supposed to pass a status field down to the tax return. In which case, I guess that makes sense. It was initially confusing to me though.

Reply:
Sorry for the confusion. I just wanted this as a convenience because I noticed that we are accumulating more scopes and logic around having a client whose tax returns are in a particular status. I wanted to make it easier to create a client with a certain status for testing.